packages/vitest/src/node/pools/rpc.ts

```diff
@@ -20,8 +26,32 @@ export function createMethodsRPC(project: WorkspaceProject): RuntimeRPC {
       const r = await project.vitenode.transformRequest(id)
       return r?.map as RawSourceMap | undefined
     },
-    fetch(id, transformMode) {
-      return project.vitenode.fetchModule(id, transformMode)
+    async fetch(id, transformMode) {
```

Comment:
These will be breaking changes. Maybe a third option that would activate this fs usage? And remove that option in V2.

`createMethodsRPC` is part of public API:

https://github.com/vitest-dev/vitest/blob/8031de7db942093a2ab3fc1e6ae42fc96bfc0786/docs/advanced/pool.md?plain=1#L60-L69

Reply:
All public API is experimental

Reply:
Oh right, it states that right on top 😄 

https://github.com/vitest-dev/vitest/blob/8031de7db942093a2ab3fc1e6ae42fc96bfc0786/docs/advanced/pool.md?plain=1#L3-L5